locust/test/test_dispatch.py

```diff
@@ -1911,6 +1912,45 @@ def test_dispatch_users_to_3_workers(self):
             self.assertTrue(0 <= delta <= 0.02, delta)
 
 
+class TestDispatchUsersToWorkersFuzzy(unittest.TestCase):
+    def test_dispatch_users_to_workers(self):
+        """
+        This "fuzzy" test uses the "dispatch_users" with various random
+        input parameters to validate that the dispatch logic does not get stuck
+        in some infinite loop or other unforeseen situations.
+        """
+
+        for _ in range(20):
+            for max_prior_users in [0, 3, 5]:
+                number_of_user_classes = random.randint(1, 30)
```

Comment:
using random in tests is not really a good idea. Just pick some values and stick with them.